egress/src/lib.rs

```diff
@@ -141,7 +163,40 @@ async fn latest_handler(
     Ok(Json(LatestResp { data }))
 }
 
-pub async fn run(pool: PgConnectionPool, s3_bucket: S3Bucket, cancel_token: CancellationToken) {
+async fn filter_handler(
+    State(pool): State<PgConnectionPool>,
+    State(filter_table): State<FilterTimeseriesTableLock>,
+    Path((station_id, param_id, sensor, level)): Path<(i32, i32, i32, i32)>,
+    Query(params): Query<FilterParams>,
```

Comment:
How is this implemented in Frost/ODA? Should we allow lists for these path params?

Reply:
Good point, it does complicate things though so perhaps its for a future refactor when we see the need for it?

Reply:
Frost is definetly the main user here, and it should be able to paralize. But we have often seen that we get better performance from one larger call, due to network overhead.

Reply:
I think what we need to address is passing a list of permits into the endpoint, so that we can use them for filtering. That is needed for the current Frost use case.